docs/scenarios/README.md

```diff
@@ -20,9 +19,48 @@ Scenarios are divided into one or more steps. Each step has an image, scenario d
 
 After the expected result, proceed to next step.
 
+## Scenario IDs
+
+Every scenario is assigned a unique ID.
+
+**`road segments`-`lanes`-`stop signs`-`scenario category`-`Ego action`-`other actors`**
+
+:::tip Example
+`2-2-XX-CW-STR-PED:N>S:01`
+:::
+
+* **road segments:** The # of road segments (arms)
+<br>2-way `2`, 3-way `3`, 4-way `4`, n/a `XX`, etc
+
+* **\# of lanes:** The # of total lanes and if there's a median/island
+<br>`1`, `2`, `3`, `4`, 2 lanes + median `2M`, 4 lanes + median `4M`, 1 lane + island in center `1I` etc
+
+* **stop signs:** Based on a bird's eye view of the scenario, where are the stop signs.
+<br>North `N`, East `E`, South `S`, West `W`, none `XX`, etc
+
+* **scenario category:** The high-level category
+<br>Car following `CF`, crosswalks `CW`, cul-de-sacs `CDS`, intersections `I`, pedestrians in road `PIR`, reversing vehicles `RV`, speed limits `SL`, vehicles in roadway `VR`, etc
+
+* **Ego action:** What Ego will be doing in the scenario
+<br>Straight `STR`, left `L`, right `R`, u-turn `U`, etc
+
+* **other actors:** The other actors in the scenario: actors' starting > ending positions (if applicable): actor actions (if applicable)
+<br>`actor:start>end:action`  
+<br>**actor:** Car `CAR`, bus `BUS`, bicycle `BIKE`, motorcycle `M`, golf cart `GC`, pedestrian `PED`, etc <br>
```

Comment:
How are multiple actors represented? How are multiple actors in different locations represented?

Reply:
Currently added to the end of the scenario ID, so every scenario ID has at minimum 6 sections (and if it has more than 6 sections, all the additional sections are just additional actors). Sort order for actors is based on the order in which Ego "encounters" the actors.

Hopefully this makes sense and is a system we can use going forward?